### DISCUSSION TITLE
Recursive login loop

### DISCUSSION THREAD

**Discussion by mfdezgallego** [apache/superset#25736] [Q&A / Help] [answered]
Hello everyone,

I have set up a Superset installation in a Rancher environment. The installation was done using Helm, following the official Superset documentation at: https://superset.apache.org/docs/installation/running-on-kubernetes

I haven't encountered any issues during the installation, and everything appears to be running correctly. I've configured an ingress to access Superset via our machine's domain. In my PC's hosts file, I've added the machine's IP and the domain.

I can access the Superset login page, but the problem arises when I enter my credentials. The login page reloads, and I can't access Superset.

Upon inspecting the pod's logs, I see the following error: flask_wtf.csrf.CSRFError: 400 Bad Request: The CSRF session token is missing.

I've tried setting the SERVER_NAME variable with the domain and another attempt with our machine's IP, but it's not working for me.

Could you please assist me with this issue?

If any additional clarification is needed, I will include it in this discussion.

Thank you very much, and best regards,
Miriam

**Comment by sfirke** [comment, marked as answer]
This has hopefully been addressed by #26005 which should be included in 3.1.0.  But people installing versions 3.0.0 - 3.0.2 or upgrading to those versions will still be affected by this.

Please see the bottom half of the discussion at https://github.com/apache/superset/issues/24579 about this problem and try the fixes outlined there.  Specifically, if you do not have HTTPS configured, try adding `"session_cookie_secure": False` to your Talisman CSP.
